ingests/ihme_child_mortality.py

```diff
@@ -3,6 +3,7 @@
 import tempfile
 import zipfile
 from pathlib import Path
+from typing import List
 
```

Comment:
If you are planning to update these datasets soon, it would be good keep the ingest scripts in a folder with their version, e.g. `ingests/ihme/2022-10-13/child_mortality.py`. This way, next time there is an update, you can simply duplicate the folder and edit the new one.